install/generator/03-syndesis-ui.yml.mustache

```diff
@@ -127,21 +127,15 @@
         },
 {{#Productized}}
         "branding": {
-          "logoWhiteBg": "",
-          "logoDarkBg": "",
-          "iconWhiteBg": "assets/images/FuseOnlineLogo_Black.svg",
-          "iconDarkBg": "assets/images/FuseOnlineLogo_White.svg",
+          "logoDarkBg": "assets/images/syndesis_logo_icon_darkbkg.png",
```

Comment:
This should be set to the product image logo, not the upstream one.  This particular file is special as it's the install template.

Reply:
Removed `"logoDarkBg": "assets/images/syndesis_logo_icon_darkbkg.png",`